index.css

```diff
@@ -92,6 +94,29 @@ button {
   transition: all 0.3s ease;
 }
 
+.error-input {
+  background-color: var(--error-bkg);
+  color: var(--brand-dark);
+  border-color: var(--brand-dark);
+}
+.error-input::placeholder {
+  color: var(--error-placeholder);
+}
+.error-label {
+  color: var(--error-dark);
```

Comment:
This is a mistake in my code! (Sorry 😅)

This CSS property should actually be `color: var(--error-dark);`